qoqo_aqt/docs/requirements.txt

```diff
@@ -1,10 +1,10 @@
-qoqo>=1.0.0
-qoqo_calculator_pyo3>=1.0.0
+qoqo>=1.2.5
+qoqo_calculator_pyo3>=1.1.0
 numpy
 sphinx >= 2.1
 nbsphinx
 pygments
 recommonmark
 myst_parser
 sphinx_rtd_theme
-qoqo_aqt>=0.5.1
+qoqo_aqt>=0.6.1
```

Comment:
Sticking to `1.2`, `1.1` and `0.6` should be enough if you want.

Reply:
Thank you!